### PR TITLE
Table-Design-UI

### PR DESCRIPTION
**Pull request by keerthanaIT**
@Neha  Done the table design, please review  this.

### CODE REVIEW

File: Table.html [hunk 1]
```diff
@@ -0,0 +1,246 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title>Table</title>
+    <style>
+        body{
+            font-family: sans-serif;
```
Comment (Neha) [inline Table.html]: Move the CSS into the separate CSS file.

File: Table.html [hunk 2]
```diff
@@ -0,0 +1,246 @@
...
+            border: none;
+        }
+
+        .scroll-table {
+            overflow-y: auto;
+            height: 250px;
+        }
+
+        .hover-table td:hover {
+            background-color: #e6e8e8 !important;
```
Comment (Neha) [inline Table.html]: Remove the !important tag

File: Table.html [hunk 3]
```diff
@@ -0,0 +1,246 @@
...
+        body{
+            font-family: sans-serif;
+            background-color: aliceblue;
+        }
+        section {
+            position: relative;
+            border: 1px solid #cac6c6;
+            padding-top: 37px;
+            background: #82dede;
+            margin-left: 100px;
```
Comment (Neha) [inline Table.html]: You can use the shorthand property for the margin property. Margin: 0 100px;

File: Table.html [hunk 4]
```diff
@@ -0,0 +1,246 @@
...
+        }
+
+        td {
+            border-left: 1px solid #eee;
+        }
+
+        td,
+        th {
+            border-bottom: 1px solid #eee;
+            background: white;
```
Comment (Neha) [inline Table.html]: For colors either have hexadecimal or the string value. Eg either use #fff or white.

File: Table.html [hunk 5]
```diff
@@ -0,0 +1,246 @@
...
+            color: #000;
+            padding: 10px 25px;
+        }
+
+        th {
+            height: 0;
+            line-height: 0;
+            padding-top: 0;
+            padding-bottom: 0;
```
Comment (Neha) [inline Table.html]: For padding use shorthand property.

File: Table.html [hunk 6]
```diff
@@ -0,0 +1,246 @@
...
+
+        th div {
+            position: absolute;
+            background: transparent;
+            color: #fff;
+            padding: 9px 25px;
+            top: 0;
+            margin-left: -25px;
+            line-height: normal;
+            border-left: 1px solid #AAD9FD;
```
Comment (Neha) [inline Table.html]: Always use lowercase for the color code.

File: Table.html (hunk 1 above)
Comment (keerthanaIT) [reply Table.html]: @Neha  Done

File: Table.html (hunk 2 above)
Comment (keerthanaIT) [reply Table.html]: Done

File: Table.html (hunk 3 above)
Comment (keerthanaIT) [reply Table.html]: Done

File: Table.html (hunk 4 above)
Comment (keerthanaIT) [reply Table.html]: Done

File: Table.html (hunk 5 above)
Comment (keerthanaIT) [reply Table.html]: Done

File: Table.html (hunk 6 above)
Comment (keerthanaIT) [reply Table.html]: Done

File: Table.html [hunk 7]
```diff
@@ -2,81 +2,8 @@
...
 <head>
+    <link rel="stylesheet" href="table.css">
     <title>Table</title>
-    <style>
-        body{
-            font-family: sans-serif;
-            background-color: aliceblue;
-        }
-        section {
-            position: relative;
```
Comment (Neha) [inline Table.html]: Hello,

1) Create a folder css-table and put the HTML and CSS file in that.
2) The approach you have taken to make the head fix can be simplify. All you need to do is remove postion from main section and divs in th. Make the thead fixed only.
3) You have declared the background color to the main section. Again it is not required, you can set that too the thead.
4) Hover effect should be on whole row.

Comment (keerthanaIT) [reply Table.html]: Hi,
ok , let me do that changes .

Comment (keerthanaIT) [reply Table.html]: Hi,
   I have Done with the all the above changes , please review it once .

File: css-table/table.css [hunk 8]
```diff
@@ -0,0 +1,72 @@
+body {
+	font-family: sans-serif;
+	background-color: #f0f8ff;
+}
+tbody {
+	display: block;
```
Comment (Neha) [inline css-table/table.css]: You can remove it. It is not required

File: css-table/table.css [hunk 9]
```diff
@@ -0,0 +1,72 @@
...
+	height: 250px;
+}
+tr {
+	display: table; 
+	width: 100%;
+  }
+h3 {
+	color: #808080;
+}
+/* .hover-table .hover-effect tr:hover{
```
Comment (Neha) [inline css-table/table.css]: Remove the comments.

File: css-table/Table.html [hunk 10]
```diff
@@ -0,0 +1,173 @@
...
+                        <td>name1@gmail.com</td>
+                        <td>123456</td>
+                    </tr>
+                    <tr>
+                        <td>name2</td>
+                        <td>name2@gmail.com</td>
+                        <td>123456</td>
+                    </tr>
+                    <tr>
+                        <td height="50">name3</td>
```
Comment (Neha) [inline css-table/Table.html]: Remove the inline-height from all places

File: css-table/table.css (hunk 9 above)
Comment (keerthanaIT) [reply css-table/table.css]: Done

File: css-table/Table.html (hunk 10 above)
Comment (keerthanaIT) [reply css-table/Table.html]: Done

File: css-table/table.css (hunk 8 above)
Comment (keerthanaIT) [reply css-table/table.css]: Hi @Neha 
    If I remove the tbody css property , then fixed header will not work , Now, do i want to remove this css property and try it in different way to fix the header ?

Thanks,
keerthanaIT :-)

Comment (keerthanaIT) [reply css-table/table.css]: Done

**Comment by keerthanaIT** [comment]
Hi @Neha 

         Done with All the changes , Please review it once .

**Comment by keerthanaIT** [comment]
Hi @Neha
       Done with All the changes , Please review it once .
Thanks,
KeerthanaIT :-)

**Comment by keerthanaIT** [comment]
Hi @Neha
            Done with All the changes , Please review it once .


Thanks,
KeerthanaIT :-)

File: css-table/table.css (hunk 8 above)
Comment (Neha) [reply css-table/table.css]: > Hi @Neha
> Done with All the changes , Please review it once .
> 
> Thanks,
> KeerthanaIT :-)

Thank you @keerthanaIT , it looks good. I hope you learned something new from this task :)

**Comment by Neha** [comment]
Thank you @keerthanaIT , it looks good. I hope you learned something new from this task :)

**Comment by keerthanaIT** [comment]
Hi @Neha 
   Thanks :-) , Yes, I learned few  things from this task. :-).

**Pull request merged**